.idea/checkstyle-idea.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These all look like default project settings. I don't think they are needed in the actual repo. Could you remove them and add .idea to the gitignore.